sample/src/main/java/com/uber/autodispose/sample/JavaActivity.java

```diff
@@ -89,34 +72,17 @@
     // Using automatic disposal, this should determine that the correct time to
     // dispose is onPause (the opposite of onResume).
     Observable.interval(1, TimeUnit.SECONDS)
-        .doOnDispose(new Action() {
-          @Override public void run() {
-            Log.i(TAG, "Disposing subscription from onResume()");
-          }
-        })
-        // `.<Long>forObservable` is necessary if you're compiling on JDK7 or below.
-        // If you're using JDK8+, then you can safely remove it.
-        .as(AutoDispose.<Long>autoDisposable(AndroidLifecycleScopeProvider.from(this)))
-        .subscribe(new Consumer<Long>() {
-          @Override public void accept(Long num) {
-            Log.i(TAG, "Started in onResume(), running until in onPause(): " + num);
-          }
-        });
+        .doOnDispose(() -> Log.i(TAG, "Disposing subscription from onResume()"))
+        .as(autoDisposable(AndroidLifecycleScopeProvider.from(this)))
+        .subscribe(num -> Log.i(TAG, "Started in onResume(), running until in onPause(): " + num));
 
     // Setting a specific untilEvent, this should dispose in onDestroy.
     Observable.interval(1, TimeUnit.SECONDS)
-        .doOnDispose(new Action() {
-          @Override public void run() {
-            Log.i(TAG, "Disposing subscription from onResume() with untilEvent ON_DESTROY");
-          }
-        })
-        .as(AutoDispose.<Long>autoDisposable(
-            AndroidLifecycleScopeProvider.from(this, Lifecycle.Event.ON_DESTROY)))
-        .subscribe(new Consumer<Long>() {
-          @Override public void accept(Long num) {
-            Log.i(TAG, "Started in onResume(), running until in onDestroy(): " + num);
-          }
-        });
+        .doOnDispose(
+            () -> Log.i(TAG, "Disposing subscription from onResume() with untilEvent ON_DESTROY"))
+        .as(autoDisposable(AndroidLifecycleScopeProvider.from(this, Lifecycle.Event.ON_DESTROY)))
+        .subscribe(
+            num -> Log.i(TAG, "Started in onResume(), running until in onDestroy(): " + num));
```

Comment:
Weird line wrap? Same with `doOnDispose`

Reply:
This is kinda how google-java-format handles it. Not too weird IMO, java lambda syntax isn't the nicest IMO

Reply:
Interesting got it!